documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1035 @@
+        type: multus
+      source:
+        id: <source_network_id>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `pod` and `multus`.
+<2> Specify a network attachment definition for each additional {virt} network.
+<3> Required only when `type` is `multus`. Specify the namespace of the {virt} network attachment definition.
+endif::[]
+
+ifdef::ostack[]
+[start=3]
+. Create a `NetworkMap` manifest to map the source and destination networks:
++
+[source,yaml,subs="attributes+"]
+----
+$  cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: NetworkMap
+metadata:
+  name: <network_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        name: <network_name>
+        type: pod <1>
+      source:<2>
+        id: <source_network_id>
+        name: <source_network_name>
+    - destination:
+        name: <network_attachment_definition> <3>
+        namespace: <network_attachment_definition_namespace> <4>
+        type: multus
+      source:<5>
+        id: <source_network_id>
+        name: <source_network_name>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `pod` and `multus`.
+<2> You can use either the `id` _or_ the `name` parameter to specify the source network. For `id`, specify the {osp} network UUID.
+<3> Specify a network attachment definition for each additional {virt} network.
+<4> Required only when `type` is `multus`. Specify the namespace of the {virt} network attachment definition.
+<5> Same as `source` above.
+endif::[]
+
+ifdef::cnv[]
+[start=3]
+. Create a `NetworkMap` manifest to map the source and destination networks:
++
+[source,yaml,subs="attributes+"]
+----
+$  cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: NetworkMap
+metadata:
+  name: <network_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        name: <network_name>
+        type: pod <1>
+      source:
+        name: <network_name>
+        type: pod
+    - destination:
+        name: <network_attachment_definition> <2>
+        namespace: <network_attachment_definition_namespace> <3>
+        type: multus
+      source:
+        name: <network_attachment_definition>
+        namespace: <network_attachment_definition_namespace> <4>
+        type: multus
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `pod` and `multus`.
+<2> Specify a network attachment definition for each additional {virt} network.
+<3> Required only when `type` is `multus`. Specify the namespace of the {virt} network attachment definition.
```

Comment:
@RichardHoch I meant that the explanation about the 'namespace' can stay but it's still duplicated below, how about changing this to
```suggestion
<3> Required only when `type` is `multus`. The `namespace` can either be specified using the namespace property or with a name built as follows: `<network_namespace>/<network_name>`.
```
and drop `<4>`? (I think there's a confusing about the " or with a name built as follows: `<network_namespace>/<network_name>`" - it also applies here)

Reply:
Fixed.